mpt_extension_sdk/routing/routers/schedule.py

```
@@ -4,24 +4,35 @@
 
 @dataclass
 class ScheduleRouter(BaseExtensionRouter):
     """Router object for schedule handlers."""
 
-    def schedule(self, path: str, name: str) -> Callable[[RouteCallback], RouteCallback]:
-        """Register a schedule handler."""
+    def task(
+        self,
+        path: str,
+        *,
+        id: str,  # ruff:ignore[builtin-argument-shadowing] - metadata contract uses "id"
```

Ruff (0.16.1)

[error] Function argument `id` is shadowing a Python builtin

(A002)